backend/src/services/cars/cars-local-search.service.ts

```diff
@@ -47,7 +47,7 @@ const carsSearch = async (data: CarsSearchParams): Promise<SearchResult[]> => {
         {
           brand: {
             id: {
-              in: data.brandId,
+              in: data.brandId ? data.brandId : data.modelId ? [] : undefined,
```

Comment:
I don't understand ;(
Don't include cars by brand if the model is present?

Reply:
Yes, we already pass certain model of brand, and if modelId present, we don't need its brandId.
It's a little unexpected code, but I tested and it works correctly.